HMPPS EMS GitHub Actions Service Accounts Cannot Deploy ConfigMaps

```
[INFO] 2020/09/16 16:50:57 main.go:637: deploying configmap/grafana-dev
[ERROR] 2020/09/16 16:50:58 main.go:256: Error from server (Forbidden): error when retrieving current configuration of:
Resource: "/v1, Resource=configmaps", GroupVersionKind: "/v1, Kind=ConfigMap"
Name: "grafana-dev", Namespace: "hmpps-ems-dev"
from server for: "STDIN": configmaps "grafana-dev" is forbidden: User "system:serviceaccount:hmpps-ems-dev:github-actions" cannot get resource "configmaps" in API group "" in the namespace "hmpps-ems-dev"
##[error]Process completed with exit code 1.
```

--- a/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-ems-dev/05-serviceaccount.yaml
+++ b/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-ems-dev/05-serviceaccount.yaml
@@ -19,6 +19,7 @@ rules:
       - "secrets"
       - "services"
       - "pods"
+      - "configmaps"
     verbs:
       - "patch"
       - "get"

